README.md
# AuiCore

MeetingStone/CHANGELOG
80100.03
    适配随身集合石
80100.02
    调整活动默认排序
80100.01
    支持wow8.1
80000.07
    添加高级过滤功能
    优化活动排序
    修复装等显示为-1的问题
80000.06
    适配随身集合石
80000.05
    添加8.0分类数据
    移除自动邀请功能
80000.04
    适配暴雪接口的修改
    移除过滤和搜索功能
80000.02
    解决一处污染
80000.01
    适配wow8.0
70300.12
    修复燃烧王座三号Boss击杀状态不正确的问题
    修复任务分类搜索到单刷（可见）的问题
70300.11
    增加成员、等级、要求列的排序
    调整默认排序规则
    修复一些bug
70300.10
    过滤掉通过AceSerializer库的漏洞夹带干扰搜索结果的数据的活动
70300.09
    修复一些Bug
    添加字数过滤功能
    过滤掉一些第三方插件创建的广告活动
70300.08
    修复一些Bug
70300.07
    修复一些Bug
70300.06
    随身集合石功能支持
70300.05
    修复一些Bug
70300.04
    修复一些Bug
70300.03
    代码优化
70300.02
    修复：7.3新增的世界任务右键没有创建集合石活动的问题
    优化：搜索活动时卡顿的问题
70300.01
    适配wow7.3
70200.06
    优化搜索机制
70200.05
    关闭了快速申请功能
70200.04
    适配wow7.2.5
70200.03
    优化部分代码
70200.02
    支持统计新的史诗副本
70200.01
    适配WOW7.2
    修改一些BUG
70100.09
    修复一处报错
    屏蔽更多的非法创建的活动
    添加仅战网好友和公会成员可见的活动
70100.08
    随身集合石上线
    过滤一些非法创建的活动
70100.06
    修复暗夜要塞Boss不正确的bug
70100.05
    优化代码
70100.04
    自动清理超过30天并且没有备注的玩友
70100.03
    添加最近玩友功能
    添加世界任务快速申请功能
        世界Boss快速加入低于40人的活动
        其它任务快速加入低于5人的活动
    修复自动弹出的世界任务在ObjectiveTracker上不能右键点击的bug
70100.02
    修复一些偶现的报错
70100.01
    适配WOW7.1
70000.04
    低等级角色整合PVP活动类型报错的BUG
70000.03
    修复地图技能不可用
    添加地下城密境类型
    增加整合PvP活动类型的选项
    优化任务二级菜单顺序
70000.02
    添加活动形式：世界任务
    添加快速创建/搜索世界任务
70000.01
    适配WOW7.0
60200.12
    使用暴雪官方的史诗地狱火
60200.11
    修复一些可能的问题
60200.10
    兑换平台去除魔兽世界时光徽章（API已不可用）
60200.09
    添加单刷可见类型
    修复团队助手不能组人的BUG
60200.08
    添加活动形式过滤
    添加过滤配置切换器
    使用服务器搜索
    数字输入框可以使用鼠标滚轮快速调整
60200.07
    预组队搜索界面优化
    预组队创建界面优化
    修复一些bug
60200.06
    集合石添加史诗地狱火堡垒
    集合石添加单刷活动
    优化集合石活动在预组队中的标题
    优化了活动排序
    客户端非前台时，有新活动申请闪烁任务栏
60200.05
    屏蔽支付宝团
60200.04
    修复服务器名带括号的活动显示不正确的问题
60200.03
    添加多官方活动
60200.02
    修复部分团队副本经验不显示的问题
60200.01
    适配WOW6.2
60100.07
    兑换平台增加魔兽世界时光徽章
60100.06
    为后续活动做准备
60100.05
    设置面板增加只显示集合石活动
    增加活动列表关键字过滤
60100.04
    修复按键绑定
    性能优化
60100.03
    增加活动平台
60100.02
    修复无法搜索部分活动类型
    屏蔽列表增强兼容 IsIgnored API.
60100.01
    修复无法创建活动的BUG
    增加活动列表高级过滤
60000.10
    增加申请列表自动邀请
    增加系统屏蔽列表增强
    创建活动时数据增加团长名字段
    优化队伍名称格式
    修复申请列表鼠标提示错误
60000.09
    记录最近5条输入的过滤条件
    团员的管理活动页增加活动说明
    修复一些bug
60000.08
    增加了小地图图标
    增加了最近搜索列表
    增加了最近创建列表
    增加了设置界面
    修复一些bug
60000.07
    与预创建队伍兼容
    集合石自动更新转变成手动刷新
    支持小队加入
    支持团长助理管理申请列表
    不符合要求的玩家将无法看到活动
    优化了创建界面和申请列表界面
    暂时取消快速组团功能
    暂时取消易信相关功能
60000.05
    调整活动类型及形式
    修复一些bug
60000.04
    重写通信协议
    增加快速申请功能
    优化公告改版：让用户不会错过我们的重要公告
    优化关注团长：现在会记录参加过的活动的团长
    优化团长副本进度：BOSS名|cff00ff00绿色|r未击杀，|cffff0000红色|r已击杀
    优化申请留言：现在可以自行填写
    优化创建活动：限制PvP等级及装备等级
    优化插件图标：现在为蓝色的集合石图标
    优化插件界面
60000.03
    解决与部分整合插件冲突的问题
60000.02
    修复6.0被邀请报错的问题
60000.01
    更新 WoW6.0
    添加暂停招募成员的功能
50400.20
    战网积分兑换平台进驻集合石插件
50400.19
    右击团长跨服密语功能
    暴雪兑换平台功能
    活动信息推送功能
50400.18
    优化数据交互，插件更加流畅
    正确提示试玩帐号不可用
    增加活动右键菜单功能
    增加活动举报功能
    增加更新日志
    增加新手帮助
    修复一些Bug
50400.17
    集合石聊天模块调试中，暂时禁用
    细化查找分类，更进一步优化组队类型
    我的队伍界面增加直接关注团长功能
    修复悬浮窗无法隐藏问题
    修复由于语言过滤插件造成的卡顿问题
    修复其他小bug
50400.16
    友团插件更名为集合石插件
    功能调整：
        增加关注团长功能
        增加最近组队功能
        增加活动分类(任务，弹性奥格的四个区)
        优化活动过滤(可用二级选项过滤，如:熊猫人之谜)
        团队通告增加职责
    UI调整：
        增加创建活动界面数字输入框增加+-按钮及滚轮调整
        优化查找活动界面人数显示
        增加小地图上右键功能菜单
        各种UI优化
50400.15
    增加活动详细介绍
    增加申请团队时的留言
    优化UI设计
    修复部分BUG
50400.14
    修复一些bug
    官方集合石开启
        1，集合石活动插件内浏览
        2，集合石活动一键组队
    加入推荐人功能
    主题活动重构为本周悬赏
    易信公众迁移到魔兽世界公众号
50400.13
    添加官方认证系统
    修改职责系统:
        现在与Blizzard职责无关，我的队伍面板将显示玩家申请的职责
    增加签到和推荐人功能
    增加活动类型: 斡耳朵斯 四大天神 世界Boss一波流
    美化一些图标
50400.12
    全部活动类型可跨服
    创建活动默认开启跨服
    优化邀请模块
    修复删除原战网好友
    修复Bagnon信息窗口兼容问题
    官网论坛开启
50400.11
    新增透明悬浮窗
    新增插件设置菜单
    新增网站活动标签
    新增组团插件与聊天插件相互切换的入口
    新增广播活动与广播插件功能
    修复了一些职责数量的显示问题
50400.10
    重新整理活动分类
    重新设计活动形式，与活动类型相关联
    添加团长PVP等级显示
    添加记录最后打开的标签
    修复删除原战网好友
    整合黑名单及操作记录标签
50400.09
    添加黑名单功能
    添加成员间查看装备属性
    规范通信频道名称
    修复观察异常问题
    调整职责设置
    优化数据交互
    修复查找活动人员显示异常问题


MeetingStone/ChangeLog.lua
BuildEnv(...)

ADDON_SUMMARY = [[
<html>
<body>
<br/>
<h2>◆  跨全区服组队平台</h2>
<h2>◆  装等，评级等级，副本经验直接查看</h2>
<h2>◆  与预创建队伍无缝兼容，|cffff1919但不建议同时使用|r</h2>
<h2>◆  由于游戏系统的限制，活动列表有|cffff1919显示上限|r</h2>
<br/>
<br/>
<h2>|cff00ff00新功能：|r 世界任务快速申请</h2>
<h2>|cff00ff00新功能：|r 记录你的最近玩友</h2>
<h2>|cff00ffff功能调整：|r 单刷活动增加可见类型</h2>
<h2>|cff00ffff功能调整：|r 服务器屏蔽支付宝活动</h2>
</body>
</html>]]

ADDON_CHANGELOG = [[
<html>
<body>
<h3>更新日志</h3>
<br/>
<h1>80100.03</h1>
<p>　适配随身集合石</p>
<br/>
<h1>80100.02</h1>
<p>　调整活动默认排序</p>
<br/>
<h1>80100.01</h1>
<p>　支持wow8.1</p>
<br/>
<h1>80000.07</h1>
<p>　添加高级过滤功能</p>
<p>　优化活动排序</p>
<p>　修复装等显示为-1的问题</p>
<br/>
<h1>80000.06</h1>
<p>　适配随身集合石</p>
<br/>
<h1>80000.05</h1>
<p>　添加8.0分类数据</p>
<p>　移除自动邀请功能</p>
<br/>
<h1>80000.04</h1>
<p>　适配暴雪接口的修改</p>
<p>　移除过滤和搜索功能</p>
<br/>
<h1>80000.02</h1>
<p>　解决一处污染</p>
<br/>
<h1>80000.01</h1>
<p>　适配wow8.0</p>
<br/>
<h1>70300.12</h1>
<p>　修复燃烧王座三号Boss击杀状态不正确的问题</p>
<p>　修复任务分类搜索到单刷（可见）的问题</p>
<br/>
<h1>70300.11</h1>
<p>　增加成员、等级、要求列的排序</p>
<p>　调整默认排序规则</p>
<p>　修复一些bug</p>
<br/>
<h1>70300.10</h1>
<p>　过滤掉通过AceSerializer库的漏洞夹带干扰搜索结果的数据的活动</p>
<br/>
<h1>70300.09</h1>
<p>　修复一些Bug</p>
<p>　添加字数过滤功能</p>
<p>　过滤掉一些第三方插件创建的广告活动</p>
<br/>
<h1>70300.08</h1>
<p>　修复一些Bug</p>
<br/>
<h1>70300.07</h1>
<p>　修复一些Bug</p>
<br/>
<h1>70300.06</h1>
<p>　随身集合石功能支持</p>
<br/>
<h1>70300.05</h1>
<p>　修复一些Bug</p>
<br/>
<h1>70300.04</h1>
<p>　修复一些Bug</p>
<br/>
<h1>70300.03</h1>
<p>　代码优化</p>
<br/>
<h1>70300.02</h1>
<p>　修复：7.3新增的世界任务右键没有创建集合石活动的问题</p>
<p>　优化：搜索活动时卡顿的问题</p>
<br/>
<h1>70300.01</h1>
<p>　适配wow7.3</p>
<br/>
<h1>70200.06</h1>
<p>　优化搜索机制</p>
<br/>
<h1>70200.05</h1>
<p>　关闭了快速申请功能</p>
<br/>
<h1>70200.04</h1>
<p>　适配wow7.2.5</p>
<br/>
<h1>70200.03</h1>
<p>　优化部分代码</p>
<br/>
<h1>70200.02</h1>
<p>　支持统计新的史诗副本</p>
<br/>
<h1>70200.01</h1>
<p>　适配WOW7.2</p>
<p>　修改一些BUG</p>
<br/>
<h1>70100.09</h1>
<p>　修复一处报错</p>
<p>　屏蔽更多的非法创建的活动</p>
<p>　添加仅战网好友和公会成员可见的活动</p>
<br/>
<h1>70100.08</h1>
<p>　随身集合石上线</p>
<p>　过滤一些非法创建的活动</p>
<br/>
<h1>70100.06</h1>
<p>　修复暗夜要塞Boss不正确的bug</p>
<br/>
<h1>70100.05</h1>
<p>　优化代码</p>
<br/>
<h1>70100.04</h1>
<p>　自动清理超过30天并且没有备注的玩友</p>
<br/>
<h1>70100.03</h1>
<p>　添加最近玩友功能</p>
<p>　添加世界任务快速申请功能</p>
<p>　　世界Boss快速加入低于40人的活动</p>
<p>　　其它任务快速加入低于5人的活动</p>
<p>　修复自动弹出的世界任务在ObjectiveTracker上不能右键点击的bug</p>
<br/>
<h1>70100.02</h1>
<p>　修复一些偶现的报错</p>
<br/>
<h1>70100.01</h1>
<p>　适配WOW7.1</p>
<br/>
<h1>70000.04</h1>
<p>　低等级角色整合PVP活动类型报错的BUG</p>
<br/>
<h1>70000.03</h1>
<p>　修复地图技能不可用</p>
<p>　添加地下城密境类型</p>
<p>　增加整合PvP活动类型的选项</p>
<p>　优化任务二级菜单顺序</p>
<br/>
<h1>70000.02</h1>
<p>　添加活动形式：世界任务</p>
<p>　添加快速创建/搜索世界任务</p>
<br/>
<h1>70000.01</h1>
<p>　适配WOW7.0</p>
<br/>
<h1>60200.12</h1>
<p>　使用暴雪官方的史诗地狱火</p>
<br/>
<h1>60200.11</h1>
<p>　修复一些可能的问题</p>
<br/>
<h1>60200.10</h1>
<p>　兑换平台去除魔兽世界时光徽章（API已不可用）</p>
<br/>
<h1>60200.09</h1>
<p>　添加单刷可见类型</p>
<p>　修复团队助手不能组人的BUG</p>
<br/>
<h1>60200.08</h1>
<p>　添加活动形式过滤</p>
<p>　添加过滤配置切换器</p>
<p>　使用服务器搜索</p>
<p>　数字输入框可以使用鼠标滚轮快速调整</p>
<br/>
<h1>60200.07</h1>
<p>　预组队搜索界面优化</p>
<p>　预组队创建界面优化</p>
<p>　修复一些bug</p>
<br/>
<h1>60200.06</h1>
<p>　集合石添加史诗地狱火堡垒</p>
<p>　集合石添加单刷活动</p>
<p>　优化集合石活动在预组队中的标题</p>
<p>　优化了活动排序</p>
<p>　客户端非前台时，有新活动申请闪烁任务栏</p>
<br/>
<h1>60200.05</h1>
<p>　屏蔽支付宝团</p>
<br/>
<h1>60200.04</h1>
<p>　修复服务器名带括号的活动显示不正确的问题</p>
<br/>
<h1>60200.03</h1>
<p>　添加多官方活动</p>
<br/>
<h1>60200.02</h1>
<p>　修复部分团队副本经验不显示的问题</p>
<br/>
<h1>60200.01</h1>
<p>　适配WOW6.2</p>
<br/>
<h1>60100.07</h1>
<p>　兑换平台增加魔兽世界时光徽章</p>
<br/>
<h1>60100.06</h1>
<p>　为后续活动做准备</p>
<br/>
<h1>60100.05</h1>
<p>　设置面板增加只显示集合石活动</p>
<p>　增加活动列表关键字过滤</p>
<br/>
<h1>60100.04</h1>
<p>　修复按键绑定</p>
<p>　性能优化</p>
<br/>
<h1>60100.03</h1>
<p>　增加活动平台</p>
<br/>
<h1>60100.02</h1>
<p>　修复无法搜索部分活动类型</p>
<p>　屏蔽列表增强兼容 IsIgnored API.</p>
<br/>
<h1>60100.01</h1>
<p>　修复无法创建活动的BUG</p>
<p>　增加活动列表高级过滤</p>
<br/>
<h1>60000.10</h1>
<p>　增加申请列表自动邀请</p>
<p>　增加系统屏蔽列表增强</p>
<p>　创建活动时数据增加团长名字段</p>
<p>　优化队伍名称格式</p>
<p>　修复申请列表鼠标提示错误</p>
<br/>
<h1>60000.09</h1>
<p>　记录最近5条输入的过滤条件</p>
<p>　团员的管理活动页增加活动说明</p>
<p>　修复一些bug</p>
<br/>
<h1>60000.08</h1>
<p>　增加了小地图图标</p>
<p>　增加了最近搜索列表</p>
<p>　增加了最近创建列表</p>
<p>　增加了设置界面</p>
<p>　修复一些bug</p>
<br/>
<h1>60000.07</h1>
<p>　与预创建队伍兼容</p>
<p>　集合石自动更新转变成手动刷新</p>
<p>　支持小队加入</p>
<p>　支持团长助理管理申请列表</p>
<p>　不符合要求的玩家将无法看到活动</p>
<p>　优化了创建界面和申请列表界面</p>
<p>　暂时取消快速组团功能</p>
<p>　暂时取消易信相关功能</p>
<br/>
<h1>60000.05</h1>
<p>　调整活动类型及形式</p>
<p>　修复一些bug</p>
<br/>
<h1>60000.04</h1>
<p>　重写通信协议</p>
<p>　增加快速申请功能</p>
<p>　优化公告改版：让用户不会错过我们的重要公告</p>
<p>　优化关注团长：现在会记录参加过的活动的团长</p>
<p>　优化团长副本进度：BOSS名|cff00ff00绿色|r未击杀，|cffff0000红色|r已击杀</p>
<p>　优化申请留言：现在可以自行填写</p>
<p>　优化创建活动：限制PvP等级及装备等级</p>
<p>　优化插件图标：现在为蓝色的集合石图标</p>
<p>　优化插件界面</p>
<br/>
<h1>60000.03</h1>
<p>　解决与部分整合插件冲突的问题</p>
<br/>
<h1>60000.02</h1>
<p>　修复6.0被邀请报错的问题</p>
<br/>
<h1>60000.01</h1>
<p>　更新 WoW6.0</p>
<p>　添加暂停招募成员的功能</p>
<br/>
<h1>50400.20</h1>
<p>　战网积分兑换平台进驻集合石插件</p>
<br/>
<h1>50400.19</h1>
<p>　右击团长跨服密语功能</p>
<p>　暴雪兑换平台功能</p>
<p>　活动信息推送功能</p>
<br/>
<h1>50400.18</h1>
<p>　优化数据交互，插件更加流畅</p>
<p>　正确提示试玩帐号不可用</p>
<p>　增加活动右键菜单功能</p>
<p>　增加活动举报功能</p>
<p>　增加更新日志</p>
<p>　增加新手帮助</p>
<p>　修复一些Bug</p>
<br/>
<h1>50400.17</h1>
<p>　集合石聊天模块调试中，暂时禁用</p>
<p>　细化查找分类，更进一步优化组队类型</p>
<p>　我的队伍界面增加直接关注团长功能</p>
<p>　修复悬浮窗无法隐藏问题</p>
<p>　修复由于语言过滤插件造成的卡顿问题</p>
<p>　修复其他小bug</p>
<br/>
<h1>50400.16</h1>
<p>　友团插件更名为集合石插件</p>
<p>　功能调整：</p>
<p>　　增加关注团长功能</p>
<p>　　增加最近组队功能</p>
<p>　　增加活动分类(任务，弹性奥格的四个区)</p>
<p>　　优化活动过滤(可用二级选项过滤，如:熊猫人之谜)</p>
<p>　　团队通告增加职责</p>
<p>　UI调整：</p>
<p>　　增加创建活动界面数字输入框增加+-按钮及滚轮调整</p>
<p>　　优化查找活动界面人数显示</p>
<p>　　增加小地图上右键功能菜单</p>
<p>　　各种UI优化</p>
<br/>
<h1>50400.15</h1>
<p>　增加活动详细介绍</p>
<p>　增加申请团队时的留言</p>
<p>　优化UI设计</p>
<p>　修复部分BUG</p>
<br/>
<h1>50400.14</h1>
<p>　修复一些bug</p>
<p>　官方集合石开启</p>
<p>　　1，集合石活动插件内浏览</p>
<p>　　2，集合石活动一键组队</p>
<p>　加入推荐人功能</p>
<p>　主题活动重构为本周悬赏</p>
<p>　易信公众迁移到魔兽世界公众号</p>
<br/>
<h1>50400.13</h1>
<p>　添加官方认证系统</p>
<p>　修改职责系统:</p>
<p>　　现在与Blizzard职责无关，我的队伍面板将显示玩家申请的职责</p>
<p>　增加签到和推荐人功能</p>
<p>　增加活动类型: 斡耳朵斯 四大天神 世界Boss一波流</p>
<p>　美化一些图标</p>
<br/>
<h1>50400.12</h1>
<p>　全部活动类型可跨服</p>
<p>　创建活动默认开启跨服</p>
<p>　优化邀请模块</p>
<p>　修复删除原战网好友</p>
<p>　修复Bagnon信息窗口兼容问题</p>
<p>　官网论坛开启</p>
<br/>
<h1>50400.11</h1>
<p>　新增透明悬浮窗</p>
<p>　新增插件设置菜单</p>
<p>　新增网站活动标签</p>
<p>　新增组团插件与聊天插件相互切换的入口</p>
<p>　新增广播活动与广播插件功能</p>
<p>　修复了一些职责数量的显示问题</p>
<br/>
<h1>50400.10</h1>
<p>　重新整理活动分类</p>
<p>　重新设计活动形式，与活动类型相关联</p>
<p>　添加团长PVP等级显示</p>
<p>　添加记录最后打开的标签</p>
<p>　修复删除原战网好友</p>
<p>　整合黑名单及操作记录标签</p>
<br/>
<h1>50400.09</h1>
<p>　添加黑名单功能</p>
<p>　添加成员间查看装备属性</p>
<p>　规范通信频道名称</p>
<p>　修复观察异常问题</p>
<p>　调整职责设置</p>
<p>　优化数据交互</p>
<p>　修复查找活动人员显示异常问题</p>
</body>
</html>]]

